algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala

```diff
@@ -94,6 +188,8 @@ class MomentsMonoid extends Monoid[Moments] with CommutativeMonoid[Moments] {
    * Uses a more stable online algorithm which should be suitable for
    * large numbers of records similar to:
    * http://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm
+   *
+   * we no longer use this, but we can't remove it due to binary compatibility
    */
   def getCombinedMean(n: Long, an: Double, k: Long, ak: Double): Double =
```

Comment:
```suggestion
  @deprecated("Use Moments.getCombinedMeanDouble instead", since = "0.13.8")
  def getCombinedMean(n: Long, an: Double, k: Long, ak: Double): Double =
```

Reply:
@johnynek wdyt?